source/extensions/filters/http/cache/http_cache_utils.h

```diff
@@ -22,6 +22,13 @@ class HttpCacheUtils {
   // header_entry is null or malformed.
   static SystemTime httpTime(const Http::HeaderEntry* header_entry);
 
+  /**
+   * Read a leading positive decimal integer value and advance "*s" past the
+   * digits read. If overflow occurs, or no digits exist, return
+   * absl::nullopt without advancing "*str".
+   */
+  static absl::optional<uint64_t> readAndRemoveLeadingDigits(absl::string_view& str);
```

Comment:
There's one more occurrence of "*s" in the comment.

Reply:
Ah, missed that!